kyverno/authorization/protect-host-namespaces.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: protect-host-namespaces
+spec:
+  validationFailureAction: enforce
+  background: true
+  rules:
+    - name: Prevent from any scc to grant access to hosts' namespaces
+      exclude:
+        resources:
+          kinds:
+            - SecurityContextConstraints
+          name: privileged
+      match:
+        resources:
+          kinds:
+            - SecurityContextConstraints
+      validate:
+        message: "Allowing access to any of the following is forbidden: allowHostIPC/allowHostNetwork/allowHostPID/allowHostPorts"
+        pattern:
```

Comment:
I think that it might be more "correct" to separate the pattern into multiple policies with unique violation messages. In my opinion, it would be more verbose, and the user will not struggle to deploy the policy in an already running environment. 

Furthermore, disabling this policy will remove all regulations regarding `host namespaces`, while disabling a single entry in the pattern will produce less damage for the cluster security.

Reply:
what do you think? @igorkchyts